docs/sql-performance-tuning.md

```diff
@@ -342,4 +371,33 @@ Data skew can severely downgrade the performance of join queries. This feature d
        </td>
        <td>3.0.0</td>
      </tr>
+     <tr>
+       <td><code>spark.sql.adaptive.forceOptimizeSkewedJoin</code></td>
+       <td>false</td>
+       <td>
+         When true, force enable OptimizeSkewedJoin even if it introduces extra shuffle.
```

Comment:
For `OptimizeSkewedJoin`, it might be hard to understand what it means here. May we provide a little context? For example,

```suggestion
         When true, force enable OptimizeSkewedJoin, which is an adaptive rule to optimize skewed joins to avoid straggler tasks, even if it introduces extra shuffle.
```

Reply:
Thank you!